build.gradle

```diff
@@ -33,24 +31,32 @@ publishing {
     }
     publications {
         mavenJar(MavenPublication) {
-            artifact bootWar
+            artifact bootJar
         }
     }
 }
 
 bootWar {
-    launchScript()
-    dependsOn(compileGroovyPages)
+    enabled = false
 }
 
 war {
+    enabled = false
```

Comment:
Adding the bootJar artefact does not require disabling the war. Revert this `war` block change.